.gitignore

```diff
@@ -33,6 +33,7 @@ Makefile.in
 /stamp-h1
 /*.pc
 
+.vscode
```

Comment:
We don't include editor-specific configuration in `.gitignore`; please remove.  You can use the `core.excludesFile` Git setting to configure this globally on your system.